src/Xml/XmlNode.cpp

```diff
@@ -19,9 +19,9 @@ using namespace NAS2D::Xml;
 /**
  * Default c'tor.
  */
-XmlNode::XmlNode() :	XmlBase(),
-						_parent(nullptr),
-						_type(XML_UNKNOWN),
+XmlNode::XmlNode() :	XmlBase(), _parent(nullptr)
+, _type(NodeType::XML_UNKNOWN)
+,
```

Comment:
Again, the formatting on this looks particularly off.

Reply:
See previous